warp/src/main/java/com/schibsted/nmp/warp/components/legacy/WarpRadioView.kt

```diff
@@ -0,0 +1,59 @@
+package com.schibsted.nmp.warp.components.legacy
+
+import android.content.Context
+import android.util.AttributeSet
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.platform.AbstractComposeView
+import androidx.core.content.withStyledAttributes
+import com.schibsted.nmp.warp.R
+import com.schibsted.nmp.warp.components.WarpRadio
+import com.schibsted.nmp.warp.components.ext.getTextFromIdOrString
+
+class WarpRadioView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyle: Int = 0
+) : AbstractComposeView(context, attrs, defStyle) {
+
```

Comment:
Here you need to inject the theme to get the correct brand colors. 
`val theme: LegacyWarpTheme by inject(LegacyWarpTheme::class.java)`